kie-maven-plugin/src/test/java/org/kie/maven/plugin/AdditionalPropertiesIntegrationTest.java

```diff
@@ -31,11 +31,12 @@ public void testAdditionalPropertiesCorrectlySet() throws Exception {
         File basedir = resources.getBasedir("kjar-3-properties-only");
         MavenExecutionResult result = mavenRuntime
                 .forProject(basedir)
-                .execute("clean", "install");
+                .execute("clean", "install", "-X");
         result.assertErrorFreeLog();
         // additional properties are logged during debug (-X) build
         // following string is created directly inside the KIE Maven plugin execution (the property names and values
         // are logged multiple by maven itself as well, so we should check directly against that string)
+
```

Comment:
This new line should be removed as it now visually separates the comment from the line which the comment belongs to (IIUC).